RealmSwift/Tests/ObjectSchemaInitializationTests.swift

```diff
@@ -182,6 +182,17 @@ class ObjectSchemaInitializationTests: TestCase {
         let types = Set(schema.properties.map { $0.type })
         XCTAssertEqual(types, Set([.string, .string, .data, .date, .object, .int, .float, .double, .bool]))
     }
+    
+    func testImplicitlyUnwrappedOptionalProperties() {
```

Comment:
Is there a reason the Swift 2.2 test wasn't sufficient? It's important to keep differences between Swift version implementations to a minimum to facilitate the transition.

```swift
func testImplicitlyUnwrappedOptionalsAreParsedAsOptionals() {
    let schema = SwiftImplicitlyUnwrappedOptionalObject().objectSchema
    XCTAssertTrue(schema["optObjectCol"]!.optional)
    XCTAssertTrue(schema["optNSStringCol"]!.optional)
    XCTAssertTrue(schema["optStringCol"]!.optional)
    XCTAssertTrue(schema["optBinaryCol"]!.optional)
    XCTAssertTrue(schema["optDateCol"]!.optional)
}
```